src/components/SearchField.tsx

```diff
@@ -210,6 +230,16 @@ const SearchField: React.FC = () => {
                 ))}
               </select>
             </div>
+            <div className="search-field-message-wrapper">
+              {isYearDecrease && (
+                <p className="search-field-message">The lowest possible year</p>
+              )}
+              {isYearIncrease && (
+                <p className="search-field-message">
+                  A movie hasn't been released
```

Comment:
this message appropriate for toYear is over todays year...

Reply:
Could you please explain more about this one? Should I change the message itself when a user tries to change a year vary from a current one?. I thoughtt if a user can't choose a year that is greater than a current one it means a particular movie hasn't been released yet :)

Reply:
Will it be correct if i let's say create a message "Date is invalid" for both date ranges?